lib/sum_of_multiples.rb

```diff
@@ -0,0 +1,22 @@
+class SumOfMultiples
```

Comment:
Problem Statement shouldn't necessarily be class names. Look around how to name your classes properly.